plugin/pkg/admission/priority/admission_test.go

```diff
@@ -32,7 +32,7 @@ import (
 	"k8s.io/kubernetes/pkg/features"
 )
 
-func addPriorityClasses(ctrl *PriorityPlugin, priorityClasses []*scheduling.PriorityClass) {
+func setPriorityClasses(ctrl *PriorityPlugin, priorityClasses []*scheduling.PriorityClass) {
```

Comment:
Any particular reason for renaming this function? I think "add" was a better name.

Reply:
No strong reasons for this renaming.
For me:
`add` means keep both the original classes and the new classes.
`keep` means, remove the original classes, then add the new classes.

`add` is also OK for me.

Reply:
OK, I have taken the name back.